src/lib/pipeline/DispatchIssueUnit.cc

```diff
@@ -56,8 +56,8 @@ void DispatchIssueUnit::tick() {
 
   /** Stores the number of instructions dispatched for each
    * reservation station. */
-  std::vector<uint16_t> dispatches = {
-      0, static_cast<unsigned short>(reservationStations_.size())};
+  std::vector<uint16_t> dispatches(
+      static_cast<unsigned short>(reservationStations_.size()), 0);
```

Comment:
No cast needed

Reply:
Had warnings on the MacStudio for it not being cast so will be keeping casting.

Reply:
What was the warning?

Reply:
Wasn't recorded.